zoompage.php

```diff
@@ -96,6 +96,10 @@ function zoomPage($id, $urlhash)
 	if ($hideinfo == false || $correcthash == true || User::getCurrent()->isAdmin() || User::getCurrent()->isCheckuser()) {
 		$smarty->assign("showinfo", true);
 	}
+
+	$emailDomain = explode("@", $request->getEmail())[1];
+	$smarty->assign("emailurl", $emailDomain);
+	$smarty->assign("emailurlencode", urlencode($emailDomain));
```

Comment:
this urlencode() call is unnecessary with `|escape:'url'` given in the Smarty template.